public/admin/config.yml

```diff
@@ -140,8 +140,8 @@ collections:
           - name: "featured_entry"
             label: "Featured Post"
             widget: "relation"
-            collection: "blog"
-            value_field: "slug"
+            collection: "blog_posts"
```

Comment:
We use both `_` and `-` for spaces. I think we should harmonize that.

Reply:
I was trying to use `_` in the CMS config for fields related to names - This way, it's very easy to spot in our codebase that this is CMS data related. We can definitely look into harmonizing.